### ISSUE TITLE
Happymh Cloudfare blocking

### ISSUE THREAD

**Issue by strumgeistxu** [keiyoushi/tachiyomi-extensions-source#1677]
### Source information

zh.happymh 1.4.8

### Source language

Chinese 

### Steps to reproduce

Downloads get blocked by Cloudfare 

### Expected behavior

Downloads start

### Actual behavior

Downloads get blocked 

### Mihon/Tachiyomi version

0.15.3

### Android version

Android 14

### Other details

User agent is already modified and works with other sources: 
Mozilla/5.0 (Linux; Android 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Mobile Safari/537.36

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by stevenyomi** [comment]
Try some other UA strings because this source is picky about that, see #1409.

**Comment by strumgeistxu** [comment]
Thanks for replying. 
My modified UA works in WebView for this source, but when come to the download, it just doesn't work. 
Will it still be helpful to change to other UA strings?
